source/Calamari/Kubernetes/Conventions/Helm/HelmManifestAndStatusReporter.cs

```diff
@@ -47,43 +47,37 @@ await Task.Run(async () =>
                            {
                                var resourceStatusCheckIsEnabled = deployment.Variables.GetFlag(SpecialVariables.ResourceStatusCheck);
 
-                               if (
-                                   resourceStatusCheckIsEnabled
-                                   || FeatureToggle.KubernetesLiveObjectStatusFeatureToggle.IsEnabled(deployment.Variables)
-                                   || OctopusFeatureToggles.KubernetesObjectManifestInspectionFeatureToggle.IsEnabled(deployment.Variables))
+                               if (!DeploymentSupportsManifestReporting(deployment, out var reason))
                                {
```

Comment:
We now _always_ report the manifest as MI is now always enabled, so this check isn't needed anymore